test/testdata/sandbox_config.json

```diff
@@ -1,6 +1,9 @@
 {
 	"metadata": {
-		"name": "podsandbox1"
+        "name": "podsandbox1",
+        "uid": "redhat-test-ocid",
```

Comment:
@runcom Seems a tab vs space problem, just remain tab is fine, although I prefer space ; )
Updated.